Cython/Compiler/Visitor.py

```diff
@@ -833,7 +833,8 @@ def repr_of(self, node):
             elif isinstance(node, Nodes.DefNode):
                 result += "(name=\"%s\")" % node.name
             elif isinstance(node, Nodes.CFuncDefNode):
-                result += "(name=\"%s\")" % node.declared_name()
+                result += "(name=\"%s\", type=\"%s\")" % (
+                    node.declared_name(), getattr(node, "type", None))
```

Comment:
Also seems worth f-stringifying, but I guess we should rather clean up the whole method in one go, afterwards.